app/controllers/api/v1/helloworld_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::HelloworldController < ApplicationController
+  def index
+    @hello = Helloworld.find(rand(1..5))
+
+    render json: @hello
+  end
+end
```

Comment:
Hi @VICTOR-CHUKWU 
Thank you for the review.
But I don't know what went wrong on your side by running the app because on my side I can't error and everything works perfectly.
The rails server is running at: 
```
http://localhost:3000/ in your browser.
```
The react server is running at: 
```
http://localhost:3001/ in your browser.
```